Library/Homebrew/compat/requirements.rb

```diff
@@ -0,0 +1,19 @@
+require "requirements"
+
+XcodeDependency          ||= XcodeRequirement
+MysqlDependency          ||= MysqlRequirement
+PostgresqlDependency     ||= PostgresqlRequirement
+GPGDependency            ||= GPGRequirement
+TeXDependency            ||= TeXRequirement
+MercurialDependency      ||= MercurialRequirement
+GitDependency            ||= GitRequirement
+AprDependency            ||= AprRequirement
+FortranDependency        ||= FortranRequirement
+JavaDependency           ||= JavaRequirement
+JavaRequirement          ||= JavaRequirement
+LanguageModuleDependency ||= LanguageModuleRequirement
+MPIDependency            ||= MPIRequirement
+OsxfuseDependency        ||= OsxfuseRequirement
+PythonDependency         ||= PythonRequirement
+TuntapDependency         ||= TuntapRequirement
+X11Dependency            ||= X11Requirement
```

Comment:
You did everything right. The only thing you forget is to add `ConflictsWithBinaryOsxfuse ||= ConflictsWithBinaryOsxfuseRequirement`.